components/internal_server/setup.cfg

```diff
@@ -0,0 +1,35 @@
+[isort]
+known_third_party = bottle
+line_length = 120
+multi_line_output = 4
+src_paths = src
+
+[mypy]
+# The pydantic.mypy plugin fails since mypy 0.920 with this mypy error message: 
+# setup.cfg:8: error: Error importing plugin "pydantic.mypy": cannot import name 'TypeVarDef' from 'mypy.types' (/.../quality-time/components/server/venv/lib/python3.10/site-packages/mypy/types.cpython-310-darwin.so)
+# plugins = pydantic.mypy  
+html_report = build/mypy
+ignore_missing_imports = false
+incremental = false
+warn_redundant_casts = true
+warn_return_any = true
+warn_unreachable = true
+warn_unused_ignores = true
+
+[mypy-bottle]
+ignore_missing_imports = true
+
+[mypy-gevent]
+ignore_missing_imports = true
+
+[mypy-lxml.html.clean]
+ignore_missing_imports = true
```

Comment:
Does the internal server use lxml?

Reply:
I don't know, it is still in the requirements. But I wouldn't know how to check 100% certain if we still need it.